app/features-json/build_json_controller.rb

```diff
@@ -59,11 +59,8 @@ class BuildJSONController < APIController
         return
       end
 
-      branch_to_trigger = "master" # TODO: how/where do we get the default branch
-
       git_fork_config = GitForkConfig.new(
         sha: current_sha,
-        branch: branch_to_trigger,
```

Comment:
deleting this might be the right choice, but I haven't checked to see if we use this `.branch` attribute from the `git_fork_config` object anywhere.
Did you double check that?